ui/app/src/Views/Shared/Types.elm

```diff
@@ -0,0 +1,5 @@
+module Views.Shared.Types exposing (Msg(..))
+
+
+type Msg
+    = OptionalValue (Maybe String)
```

Comment:
I don't think this is needed, the msg type can be `Maybe String`.

Reply:
cool, thanks for the suggestion!